### ISSUE TITLE
Please add InforSuite Application Server as Compatible product of Jakarta EE Full Platform

### ISSUE THREAD

**Issue by Yaoz1207** [jakartaee/jakarta.ee#1071]
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://www.eclipse.org/membership/showMember.php?member_id=1487

**A product name and version**
InforSuite Application Server V10

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
https://github.com/codff/InforSuite/archive/main.zip

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
Shandong Cvicse Middleware Co.,Ltd.

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
Full Platform:https://github.com/eclipse-ee4j/jakartaee-platform/issues/306

**The URL where Compatible Software Product is available for download**
http://www.inforbus.com/as.html

**Comment by flora8984461** [comment]
Hi @codff 

Please check the preview: https://deploy-preview-1073--jakartaee.netlify.app/zh/compatibility/#tab-8 
https://deploy-preview-1073--jakartaee.netlify.app/zh/compatibility/#tab-one

We will merge to live once got +1 from you.

Thanks!

**Comment by Yaoz1207** [comment]
Hello @flora8984461 

I have checked and the information is correct. Please register to the official JakartaEE website, thank you!

**Comment by codff** [comment]
Hi @flora8984461 
I want to add another point，About"The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting"
https://www.eclipse.org/membership/editMember.php?member_id=1487

**Comment by Yaoz1207** [comment]
Hi @flora8984461 
I want to add another point，About"The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting"
https://www.eclipse.org/membership/showMember.php?member_id=1487

**Comment by chrisguindon** [comment]
> Hi @flora8984461
> I want to add another point，About"The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting"
> https://www.eclipse.org/membership/showMember.php?member_id=1487

Hi @codff ,

could you rephrase the question? We are not sure we understand your request.

**Comment by codff** [comment]
> > 您好，@ flora8984461，
> > 我想补充一点，关于“ Eclipse Foundation概要文件的URL，以确认您是您要提交其产品信息的公司的授权代表” 
> > [https://www.eclipse.org/membership/showMember .php？member_id = 1487](https://www.eclipse.org/membership/showMember.php?member_id=1487)
> 
> 嗨@codff，
> 
> 你能改一下这个问题吗？我们不确定我们是否理解您的要求。

The problem is that the first "The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting" is missing from the issue I submitted, so I want to add that the answer to this question is "https://www.eclipse.org/membership/showMember.php?member_id=1487"

**Comment by Yaoz1207** [comment]
> > Hi @flora8984461
> > I want to add another point，About"The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting"
> > https://www.eclipse.org/membership/showMember.php?member_id=1487
> 
> Hi @codff ,
> 
> could you rephrase the question? We are not sure we understand your request.

The problem is that the first "The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting" is missing from the issue I submitted, so I want to add that the answer to this question is "https://www.eclipse.org/membership/showMember.php?member_id=1487"
And I have updated the issue.

**Comment by Yaoz1207** [comment]
@chrisguindon 

Hi chrisguindon

Enter this link https://jakarta.ee/compatibility/#tab-8, And click the link "10" in the picture, the visit is wrong.
As the picture shows：
![image](https://user-images.githubusercontent.com/79249422/109240722-f3cd6200-7812-11eb-9339-c524e5c55073.png)

Please update the link,the link is"http://www.inforbus.com/InforsuiteAS/platform-tckresults/index1.html".
Thank you!

**Comment by flora8984461** [comment]
Hi @codff Thanks for the catch! I just updated it and the website will update once finished building.

Thanks!
